### PR TITLE
[Localization] Important updates to terms of use and data privacy

### PR DESCRIPTION
**Pull request by eisvogel**
Updated terms of use and data privacy statement to latest documents, both for German and English versions.

Please review these screens for both German and English:

|Terms of Use|Data Privacy Statement|
|:--|:--|
|image|image|

Attached documents represents the meta structure of the legal documents and have been signed off. The actual appearance in the app is aligned with the design typography and is not reflected.

[data-privacy-statement.de.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/4775333/data-privacy-statement.de.pdf)
[data-privacy-statement.en.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/4775334/data-privacy-statement.en.pdf)
[terms-of-use.de.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/4775335/terms-of-use.de.pdf)
[terms-of-use.en.pdf](https://github.com/corona-warn-app/cwa-app-ios/files/4775336/terms-of-use.en.pdf)

### CODE REVIEW

**Review by steins-code — DISMISSED** [review]

**Comment by ChristianKienle** [comment]
Please do not merge. development is frozen.

**Comment by EisvogelSAP** [comment]
This is a high priority escalation. Please see the ticket 1533.

**Review by ChristianKienle — APPROVED** [review]

**Comment by EisvogelSAP** [comment]
All changes have been applied.

@haosap Please approve.

CHANGES MUST NOT BE MERGED BEFORE @haosap has APPROVED THIS PR.

**Review by haosap — APPROVED** [review]

**Pull request merged**
